server/src/main/java/de/coronavirus/imis/config/SchedulingConfig.java

```diff
@@ -0,0 +1,10 @@
+package de.coronavirus.imis.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration
+@EnableScheduling
+public class SchedulingConfig {
+
+}
```

Comment:
Wofür wird das benötigt?

Reply:
Das ist noch Code, der im Zusammenhang mit der externen API von `gesetzlichekrankenkasse.de` genutzt wurde. Er war für die Aktivierung der Scheduling-Features von Spring notwendig, die zum periodischen API-Polling verwendet wurden.

In der aktuellen Form des PR wird er nicht mehr benötigt. Ich kann ihn problemlos mit einem neuen Commit rauswerfen, wenn das gewünscht ist.

Reply:
Also ich fänds cool wenn wir nicht benötigten code rausnehmen.